pkg/api/server/server.go

```diff
@@ -51,17 +56,22 @@ func WithProfiling(enabled bool) Option {
 	}
 }
 
-// WithMetrics enables prometheus metrics endpoints
-func WithMetrics(enabled bool) Option {
+// WithMetrics enables prometheus metrics endpoints. The request duration can be provided if they should differ
+// from the default duration buckets
+func WithMetrics(enabled bool, requestDurationBuckets ...float64) Option {
 	return func(server *DefaultServer) {
 		server.metrics = enabled
+		server.requestDurationBuckets = requestDurationBuckets
```

Comment:
Doesn't prometheus support a more flexible data structure for that? As you've already detected in https://github.com/els0r/goProbe/issues/74 this can easily overflow (and IMHO these are exactly the cases we want to see because they indicate unusually high latency)...

Reply:
They do, but it's experimental: Native Histograms. I will enable them eventually, but at the moment, the stack that I'm feeding the metrics into doesn't support them (yet).

I'm seriously surprised that this issue hasn't been solved earlier by the community.

Reply:
Yeah, me too. I mean, when dealing with (static) histograms, the question about under/overflow bins is one of the first things that comes up when your scale / axis isn't strictly limited. Does the current one support overflow bins (or can you add an extra bin at the end with, idk, 10m as rightmost x value that captures everything overflowing)?
Thanks for the info about native histograms though, seems like a good future option...